.travis.yml

```diff
@@ -7,6 +7,7 @@ jobs:
   include:
   - stage: Test and build
     script:
+    - rm go.sum # Workaround for https://github.com/kubernetes/kubernetes/issues/69040
```

Comment:
Travis is finally green but this ugliness was the only way I could see to achieve it.

@markfuller @thallgren What do you reckon?

Reply:
I think this is the best option at this point.